experiments/active_manifesto_experiments.py

```diff
@@ -103,19 +103,29 @@ def smooth_probas(label_probas, eps = 1e-9):
     # smoothing probabilities to avoid infs
     return np.minimum(label_probas + eps, 1.)
 
-def prioritize_samples(label_probas, strategy="margin_sampling"):
+def prioritize_samples(label_probas, strategy="margin_sampling", include=[]):
```

Comment:
Was macht das include genau?

Reply:
wir haben ja das problem, dass unser active learning sampling auch viele irrelevante, weil vermutlich 'neutrale' texte hoch priorisiert. was gut is, wenn man alle drei klassen schnell richtig vorhersagen will, aber wenn wir eher an links oder rechts interessiert sind, dachte ich, waers vielleicht besser, die klassen mehr zu beruecksichtigen und neutral hintenanzustellen beim lernen. 

hab heute mal experimente laufen lassen, sone billige heuristik hat auch irgendwie funktioniert. ich schick da gleich mal plots.

Reply:
cool!